clang/lib/CIR/Lowering/DirectToLLVM/LowerToLLVM.cpp

```diff
@@ -1133,6 +1252,7 @@ void ConvertCIRToLLVMPass::runOnOperation() {
   patterns.add<
       // clang-format off
                CIRToLLVMBrCondOpLowering,
+               CIRToLLVMBinOpLowering,
```

Comment:
This list is alphabetical (for consistency in rebasing the incubator), so this belongs one line up.